docs/API_SPECS_SIMULATIONS.md

```diff
@@ -611,10 +611,45 @@ to ignore any change happened and overwrite the existing simulation.  Note
 that this is meant to be used only when data loss is acceptable, e.g.
 during development sessions and in test environments.
 
-## Deleting a simulation
+## Delete a simulation
 
 Simulations can be deleted using the DELETE method.
 
 ```
 DELETE /v1/simulations/1
-```
\ No newline at end of file
+```
+
+## Add device in a simulation
+
+Add a device to a simulatiom using AddDevice method.
+
+Request:
+```
+PUT /v1/simulations/1/AddDevice
+Content-Type: application/json; charset=utf-8
+```
+```json
+{
+  "Item": "truck-01.0",
+}
+```
+
+## Delete devices in a simulation
+
+Devices in a simulation can be deleted using DeleteDevices method. 
+The method accepts a list of upto 100 device ids to deleted.
+
+Request:
+```
+PUT /v1/simulations/1/DeleteDevices
```

Comment:
same comment about the URL, we should avoid custom verbs and use HTTP verbs. This should be 
* DELETE /v1/devices/{id}
* POST /v1/devices!delete and the list of ids in the payload

see https://github.com/Azure/pcs-template-microservice-dotnet/blob/master/WebService/v1/Controllers/ValuesController.cs